src/Temporalio/Worker/Interceptors/CreateContinueAsNewExceptionInput.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections.Generic;
+using Temporalio.Api.Common.V1;
+using Temporalio.Workflows;
+
+namespace Temporalio.Worker.Interceptors
+{
+    /// <summary>
+    /// Input for <see cref="WorkflowOutboundInterceptor.CreateContinueAsNewException" />.
+    /// </summary>
+    /// <param name="Workflow">Workflow to continue.</param>
```

Comment:
Should we name this `WorkflowType` to disambiguate with e.g. workflow ID?
I'd at least mention in the docstring.